collections/river-and-lake-ice-extent-s1s2.yaml

```diff
@@ -0,0 +1,74 @@
+Name: River and Lake Ice Extent S1+S2
+Description: |
+  The River and Lake Ice Extent S1+S2 (RLIE S1+S2) product is part of the Copernicus Land Monitoring Service (CLMS), 
+  pan-European High Resolution Snow and Ice Monitoring [(HR-S&I)](https://land.copernicus.eu/pan-european/biophysical-parameters/high-resolution-snow-and-ice-monitoring) Ice product suite.The River and Lake Ice Extent (RLIE) products measure the presence of ice in rivers and lakes described by the EU-HYDRO network database, in particular the area of snow-free or snow-covered ice. 
+  The RLIE S1+S2 product is computed from the combination of RLIE S1 and RLIE S2 products.The RLIE S2 product from a given day is gap-filled using the RLIE S1 information from the same day. Because the RLIE S1+S2 is only produced when RLIE S2 and RLIE S1 products are aquired on the same day, it is delivered in delayed-time for the entire EEA39 zone at a spatial resolution of 20 m x 20 m.
+  More information about the RLIE S1+S2 product methodology and workflow is outlined in the [product user manual](https://land.copernicus.eu/user-corner/technical-library/hrsi-ice-pum)
+AdditionalInfoExternal:
+    Title: Additional info 
+    Path: river-and-lake-ice-extent-s1s2/README.MD 
+Image: river-and-lake-ice-extent-s1s2/thumbnail.png       
+Resolution: 20m
+GeographicalCoverage: Europe  (Lat; 26N 72N Lon; -25W 45E)
+TemporalAvailability: September 1, 2016 - ongoing 
+UpdateFrequency: Annually
```

Comment:
accroding to the descr, should be something like

`delayed time, when S1 and S2 are acquired on the same day`